cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -189,6 +189,24 @@ def _remove_docker_creds():
         view.save.click()
 
 
+@pytest.fixture(scope="module")
+def enable_smartproxy_affinity(request, appliance, provider):
+    if provider.name == 'rhv_cfme_integration':
```

Comment:
I don't think this a good approach to solving the problem at large - we should never be hard-coding solutions for our provider keys.

There needs to be a `provider.provider_data` query, and respective yaml entries added, in order to control setup of smartproxy.  It should default to False.

Reply:
Added smartproxy_affinity key to our yaml and changed the condition appropriately.